datadog/resource_datadog_synthetics_test_.go

```diff
@@ -1075,7 +1075,7 @@ func syntheticsBrowserVariableElem() *schema.Resource {
 				Default:     "",
 			},
 			"id": {
-				Description: "ID of the global variable to use. This is actually only used (and required) in the case of using a variable of type `global`.",
+				Description: "ID of the global variable to use. This is actually only used (and required) in the case of using a variable of the `global` type.",
```

Comment:
```suggestion
				Description: "ID of the global variable to use. This is actually only used (and required) in the case of using a variable of the `global` type.",
```
Just a heads up - "Of type `type`" is a common construction in the programming field.